lxd/instance_uefi_vars.go

```diff
@@ -85,6 +85,15 @@ func instanceUEFIVarsGet(d *Daemon, r *http.Request) response.Response {
 		return resp
 	}
 
+	// The act of getting UEFI vars for an instance can cause them to be generated, so ensure we lock here
+	// as well as in instanceUEFIVarsPut.
+	unlock, err := instanceOperationLock(s.ShutdownCtx, projectName, name)
```

Comment:
Do I understand correctly, that this lock will protect us from race in this case:
```
# create and start an instance
lxc launch ubuntu:22.04 v1 --vm
# do set while instance IS starting and will call `d.setupNvram` in the middle
lxc config uefi set v1 SecureBootEnable-f0a30bc7-af08-4556-99c4-001009c93a44=00
lxc config uefi get v1 SecureBootEnable-f0a30bc7-af08-4556-99c4-001009c93a44
```

So, after `lxc launch ...` command the instanceOperationLock is taken somewhere until the instance will be fully started, right?